tests/test-f.py

```diff
@@ -230,3 +230,9 @@ def test_sum():
     assert str(dt.sum(f[:])) == str(f[:].sum())
     DT = dt.Frame(A=range(1, 10))
     assert_equals(DT[:, f.A.sum()], DT[:, dt.sum(f.A)])
+
+def test_max():
+    assert str(dt.max(f.A)) == str(f.A.max())
+    assert str(dt.max(f[:])) == str(f[:].max())
+    DT = dt.Frame(A=range(1, 10))
+    assert_equals(DT[:, f.A.max()], DT[:, dt.max(f.A)])
```

Comment:
Let's add a new line at the end of this file, so that the last line becomes a valid line of code.